GATEWAY/src/providers/tilesApi.service.spec.ts

```diff
@@ -39,6 +39,10 @@ describe('tilesAPI', () => {
           provide: Storage,
           useClass: StorageMock
         },
+        {
+          provide: ViewController,
+          useClass: ViewControllerMock
+        },
```

Comment:
Not necessary
We do not need the ViewController or the ViewControllerMock, because they do not work
I have found another solution that works